### PR TITLE
[WiP] Fix AppHMIType in Policy

### PR DESCRIPTION
**Pull request by LevchenkoS**
Fixed error with missing `AppHMITypes` in `AccessRemote` component
Also fixed issue with `SQL PT`

Fix: [SDLOPEN-498](https://adc.luxoft.com/jira/browse/SDLOPEN-498)
##
@LuxoftAKutsan, please review

### CODE REVIEW

**Comment by ovvasyliev** [comment]
Rebuild required

**Comment by LevchenkoS** [comment]
@okozlovlux, please review

**Comment by ovvasyliev** [comment]
Rebuild required

File: src/components/application_manager/include/application_manager/policies/policy_handler.h [hunk 1]
```diff
@@ -440,6 +440,10 @@ class PolicyHandler : public PolicyHandlerInterface,
   virtual void OnUpdateHMILevel(const std::string& device_id,
                                 const std::string& policy_app_id,
                                 const std::string& hmi_level) OVERRIDE;
+
+  virtual void AddApplication(
```
Comment (LuxoftAKutsan) [inline src/components/application_manager/include/application_manager/policies/policy_handler.h]: @LevchenkoS remove virtual keyword

Comment (LevchenkoS) [inline src/components/application_manager/include/application_manager/policies/policy_handler.h]: @LuxoftAKutsan, fixed in 7348fb6

**Pull request merged**